src/vllm_tgis_adapter/__main__.py

```diff
@@ -245,18 +245,6 @@ async def run_http_server(
 
     engine_args = AsyncEngineArgs.from_cli_args(args)
 
-    # Enforce pixel values as image input type for vision language models
```

Comment:
Fixed in #31, should rebase on `fix-main-module` (or `main` after #31 is merged)